hmda/src/main/scala/hmda/persistence/submission/HmdaValidationError.scala

```diff
@@ -283,9 +283,8 @@ object HmdaValidationError
                 .noEditsFound()) {
             Effect.persist(signed).thenRun { _ =>
               log.info(s"Submission $submissionId signed at ${Instant.ofEpochMilli(timestamp)}")
-              updateSubmissionStatus(sharding, submissionId, Signed, log)
-              updateSubmissionReceipt(sharding, submissionId, signed.timestamp, s"${signed.submissionId}-${signed.timestamp}", log)
-              publishSignEvent(submissionId).map(signed => log.info(s"Published signed event for $submissionId"))
+              updateSubmissionStatusAndReceipt(sharding, submissionId, timestamp, s"${signed.submissionId}-${signed.timestamp}", Signed, log)
+              publishSignEvent(submissionId).map(signed => log.info(s"Published signed event for $timestamp"))
```

Comment:
Why was the log for the signed event publishing changed from including the  signed.submissionID and signed.timestamp to just the timestamp?

Reply:
unintended consequence of debugging